tests/main.nf.test

```diff
@@ -162,7 +162,7 @@ nextflow_pipeline {
                 variant_consequences_snv = params.pipelines_testdata_base_path + 'nallo/reference/variant_consequences_v2.txt'
 
                 // Parameters
-                parallel_snv    = 3
+                parallel_snv    = 2
```

Comment:
Have you changed this in the nextflow_schema?

Reply:
The default in nextflow_schema is 13. This is because chr1 fits into one region/bed, and all other regions will be smaller than chr1.